### ISSUE TITLE
Train HeadNeRF using my customized dataset

### ISSUE THREAD

**Issue by hengfei-wang** [CrisHY1995/headnerf#8]
Hi,

Thank you for releasing the code. Your work is quite impressive to me. 

I have some problems on training headnerf using my dataset. The following is the training result:

![render_img](https://user-images.githubusercontent.com/39575336/170296353-96a6e9f0-f518-4bdb-ad45-8fc841afc4c4.png)

Can you give me some advice?

Thank you!

**Comment by hengfei-wang** [comment]
I revised the learning rate, and the problem disappeared. Closed

**Comment by MingiYeom** [comment]
hi how did you train model and test without pre-train model

**Comment by prokaryote-hgy** [comment]
@hengfei-wang hi, would you please share your training code? I'm confused how to train the network.
